.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
         "eslint:recommended",
         'plugin:@typescript-eslint/recommended',
         'plugin:vue/vue3-recommended',
+        "prettier"
```

Comment:
I just followed the [instructions](https://prettier.io/docs/en/install#eslint-and-other-linters) in the prettier tutorial for users of eslint.